components/atom/accordion/README.md

```diff
@@ -0,0 +1,74 @@
+# AtomAccordion
+
+> Description
+
+<!-- ![](./assets/preview.png) -->
+
+## Installation
+
+```sh
+$ npm install @s-ui/react-atom-accordion --save
+```
+
+## Usage
+
+```js
+import AtomAccordion from '@s-ui/react-atom-accordion'
+```
+
+### Basic usage
+```js
+<AtomAccordion 
+    icon={svgIcon} 
+    showText='Show' 
+    hideText='Hide'
+>
+    <MoleculeCollapsible 
```

Comment:
Shouldn't `AtomAccordion` be `MoleculeAccordion`?
Is it ok having _atoms_ (`AtomAccordion`) using _molecules_ (`MoleculeCollapsible`) ? Shouldn't this be the opposite?